docs/planning.md

```diff
@@ -0,0 +1,81 @@
+# Release and Roadmap Process
+
+We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/open-refinement-and-planning.md).
+
+While every repository in the [eclipse-tractusx](https://github.com/eclipse-tractusx) GitHub organization
+has its own issue management, the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
+is used to align the overarching Tractus-X releases.
+
```

Comment:
I suggest adding in a new paragraph:

### Release Cycle - major vs. minor release

Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release [tractus-x-release.md](url) following CalVer is released. Each Tractus-X release contains multiple products [product_release.md](url) Once per year, Tractus-X releases a major release, making the remaining three releases minor releases. A major release may contain critical breaking changes that have a major impact on data space participants,
such as changes to enablement services. A minor release contains backward compatible functionality. In addition, patch versions provide backward compatible bug and security fixes.

Reply:
Hi @HFocken, 
I also heard that, but actually never saw that in a tangible way. 
At least "major" and "minor" is not directly visible in CalVer numbers as it would be in SemVer. Is it always the last release of a year, that will contain breaking changes? 
How is this ensured in the planning process? Do issues from open planning also contain something that makes that transparent?

Reply:
Never heard of this too. Definitly something we should highlight more in our Release Notes @RolaH1t

Reply:
use text here AND on github.io page to initially introduce users to process.
add picture of phases.